index.d.ts

```diff
@@ -0,0 +1,86 @@
+export const version: number;
+export const validOptions: string[];
+
+export type Options = Record<string, unknown>;
```

Comment:
Why not strictly type this so the user actually knows which options are valid and what value they provide for said key